cmd/hostpaths/hostpaths.go

```diff
@@ -420,6 +433,32 @@ func mapHostPaths(ctx context.Context, pManager, vManager manager.Manager) {
 	}, time.Second*5)
 }
 
+func filterManagedPods(ctx context.Context, options *context2.VirtualClusterOptions, podList *corev1.PodList, pManager manager.Manager) error {
+	if options.MultiNamespaceMode {
+		// Limit Pods
+		nsList := &corev1.NamespaceList{}
+		err := pManager.GetClient().List(ctx, nsList, &client.ListOptions{
+			LabelSelector: labels.SelectorFromSet(labels.Set{
+				namespaces.VclusterNamespaceAnnotation: options.TargetNamespace,
+			}),
+		})
+		if err != nil {
+			return errors.Wrap(err, "unable to list namespaces")
```

Comment:
```suggestion
			return fmt.Errorf("unable to list namespaces: %w", err)
```

Reply:
Just wanted to know for my understanding why fmt.Errorf("..%w", err) is favorable over errors.Wrap()

Reply:
Specifically two reasons:

- `errors.New`/`fmt.Errorf` are stdlib, while `errors.Wrap` belongs to `github.com/pkg/errors`, which is deprecated.
- In the style guide that we're adopting there is a matrix of what to use when and why: https://github.com/uber-go/guide/blob/master/style.md#errors

Reply:
Eventually, I'd like to eliminate the `github.com/pkg/errors` library in the future.